packages/block-library/src/preformatted/index.js

```diff
@@ -10,7 +10,7 @@ export const name = 'core/preformatted';
 export const settings = {
 	title: __( 'Preformatted' ),
 
-	description: __( 'Add text that respects your spacing and tabs, and also allows styling.' ),
+	description: __( 'Add custom HTML code and preview it as you edit.' ),
```

Comment:
This doesn't make sense for the Preformatted block. There is no "preview" mode; the Preformatted block is just for preformatted text.

Reply:
Good point. This seems like it wasn't updated to the latest suggestion [in this comment](https://github.com/WordPress/gutenberg/issues/10919#issuecomment-432798650):

> Add text that respects your spacing and tabs, and also allows styling.